draft-spaghetti-sidrops-rpki-ta-tiebreaker.xml

```diff
@@ -72,8 +72,8 @@ version="3">
 
     <abstract>
       <t>
-	A Trust Anchor (TA) in the RPKI is represented by a self-signed X.509 Certification Authority (CA) certificate.
-        It is possible for Relying Parties (RP) to have acquired multiple different issuances of valid TA certificates from the same TA operator.
+        A Trust Anchor (TA) in the RPKI issues a self-signed X.509 Certification Authority (CA) certificate at a well-known location.
+        Over time, Relying Parties (RP) may have acquired multiple, all different and valid,  TA certificates from the same TA.
```

Comment:
The original sentence flows better, I prefer that one

Reply:
I think the main objection is that  the new 'all different and valid,'  insertion disrupts the flow of the sentence too much. How about the simpler

`Over time, Relying Parties (RP) may have acquired multiple distinct and valid TA certificates from the same TA.`